frontends/web/src/hooks/keyboard.ts

```diff
@@ -60,7 +60,8 @@ const FOCUSABLE_SELECTOR = `
   input:not(:disabled),
   select:not(:disabled),
   textarea:not(:disabled),
-  [tabindex]:not([tabindex="-1"]):not(:disabled)
+  [tabindex]:not([tabindex="-1"]):not(:disabled),
+  [autofocus]:not(:disabled)
```

Comment:
This would still get the first input element and focus it. I believe we want either:
- all elements that either have autofocus or tabindex != -1 regardless of the tagname
- or explicitly `a[tabindex]:not([tabindex="-1"]), button[autofocus]:not(:disabled), input[autofocus]:not(:disabled)...`